### ISSUE TITLE
Clarify first paragraph of Verifier Reconciliation

### ISSUE THREAD

**Issue by yogeshbdeshpande** [ietf-rats-wg/draft-ietf-rats-corim#241]
```suggestion
While this specification focuses on the CoRIM format itself, it is also vital to  document how a Verifier should process the CoRIM so as to enable the CoRIM authors to appropriately encode it.
```

_Originally posted by @yogeshbdeshpande in https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/232#discussion_r1627880853_

**Comment by thomas-fossati** [comment]
@yogeshbdeshpande do we really need this issue?

**Comment by henkbirkholz** [comment]
resolved by PR #252
